### PR TITLE
Align this repository with official movie examples

### PR DESCRIPTION
**Pull request by fbiville**
- set up CI
 - update dependencies
 - align README with other repositories
 - simplify setup

### CODE REVIEW

**Comment by fbiville** [comment]
Note: the CI should fail because the newer vote endpoint has not been implemented.

File: index.php [hunk 1]
```diff
@@ -52,26 +58,27 @@
...
     $result = $client->run(<<<'CYPHER'
 MATCH (movie:Movie) 
-WHERE toLower(movie.title) contains toLower($title)
-RETURN {title: movie.title, released: movie.released, tagline: movie.tagline} as movie
+WHERE TOLOWER(movie.title) CONTAINS TOLOWER($title)
+RETURN movie {.title, .tagline, .votes, .released}
 CYPHER, ['title' => $request->getQueryParams()['q'] ?? '']);
 
-    $response->getBody()->write(json_encode($result, JSON_THROW_ON_ERROR));
+    $response->getBody()->write(json_encode($result->jsonSerialize()['result'], JSON_THROW_ON_ERROR));
```
Comment (fbiville) [inline index.php]: there is probably a better to serialize the result without the summary, I assume? (cc @transistive)

Comment (transistive) [reply index.php]: Yes this is an oversight of me. I have updated it here: [11572b5eb71ba4a565dddfb90ac7dc9ba14d4eae](https://github.com/neo4j-php/neo4j-php-client/commit/11572b5eb71ba4a565dddfb90ac7dc9ba14d4eae)

**Review by transistive — APPROVED** [review]
Looks all good to me!

May I ask why you moved the index.php to the root folder? It still all works, so I approve :+1:

**Comment by fbiville** [comment]
@transistive the only reason I moved the file is to make it immediately visible (in the Go example e.g., the single backend file `server.go` is at the root)

**Comment by transistive** [comment]
Good enough reason for me! Let's merge it 👍

**Pull request merged**